src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $cell-size * 7 + $gap-size * 6;
+  padding: 30px;
+  margin: 0 auto;
+
+  &--start-day {
+    @each $day, $margin in $day__names {
+      &--#{$day} :first-child {
+        margin-left: $margin * $cell-size + $margin * $gap-size;
+      }
+    }
+  }
+
+  &--month-length {
+    @for $num from 28 through 31 {
+      &--#{$num} :nth-child(n + #{$num + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+
+    background-color: #eee;
+    cursor: pointer;
+    transition-duration: 500ms;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: #f282b9;
+    }
+
+    @for $num from 1 to 32 {
+      &:nth-child(#{$num})::before {
+        content: "#{$num}";
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
You can set font family to html tag and inherit it, and set font size to `__day`

Reply:
Great idea!